geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/rebalance/PartitionedRegionLoadModelJUnitTest.java

```diff
@@ -483,30 +485,73 @@ public void testColocationEnforceLocalMaxMemory() throws UnknownHostException {
   }
 
   /**
-   * Test that each region individually honors it's enforce local max memory flag.
+   * Test that a region with enforceLocalMaxMemory disabled colocated with
+   * a region that memory full with enforceLocalMaxMemory enabled will prevent a bucket move.
    */
   @Test
-  public void testColocationIgnoreEnforceLocalMaxMemory() throws UnknownHostException {
+  public void testColocationOneEnforceLocalMaxMemoryFull() throws UnknownHostException {
```

Comment:
This seconde test case is newly added, I will explain why the third test case was changed.
Most reasons is like above, special reason for this test case is to improve the test condition.
First region lmm was set 500 before, this will make the outloop memory judege(the one I removed in this PR) pass and make the test case pass.
So I decrease the lmm of the first region, so make the total lmm not enough to hold all the data, this test case will pass with this PR applied, but will fail with the old code.